src/components/Editor/Organisations/LinkOverlay.vue

```diff
@@ -528,17 +533,29 @@
         }
       },
       watch: {
-        'menus.newOrganisation.data.logo': function(){
-          if (this.menus.newOrganisation.data.logo) {
-            let reader = new FileReader();
-            reader.addEventListener('load', async (event) => {
-              this.menus.newOrganisation.logoData = {
-                filename: this.menus.newOrganisation.data.logo.name,
-                data: event.target['result'],
-                content_type: this.menus.newOrganisation.data.logo.type
-              };
-            });
-            reader.readAsDataURL(this.menus.newOrganisation.data.logo);
+        'menus.newOrganisation.data.logo': {
+          async handler(logo) {
+            let _module = this;
+            // this.menus.newOrganisation.logoData
+            if (logo === null || logo === undefined) {
+              // This is to prevent a logo being deleted if a user fiddles about with the form and then
+              // submits with no image uploaded.
+              if (_module.organisation.urlForLogo) {
+                _module.menus.newOrganisation.delete('logoData');
+              }
+              else {
+                _module.menus.newOrganisation.logoData = {}
+              }
+              return;
+            }
+            _module.logoLoading = true;
+            let convertedFile = await toBase64(logo);
```

Comment:
Conversion toBase64 could be called on click of ```Save new organisation``` button.

Reply:
I've addressed this one here: https://github.com/FAIRsharing/fairsharing.github.io/pull/2264#discussion_r1521762363